### ISSUE TITLE
[Balance issue] Small steam per tick prodction

### ISSUE THREAD

**Issue by craft90** [elytra/BetterBoilers#18]
I decided to take this mod for a swing and I'm rather disappointed with the low steam production. I build a 3x3, 4 blocks tall boiler, hooked up a Thermal Expansion steam dynamo with the Turbine Augment (which means it requires external steam) and noticed a very low energy output. On further checking I noticed that the dynamo only received about 80 mb steam per tick. So I upped the size to a full 5x5x5 cuboid and behold it received only about 51 mb/steam, very briefly jumping up to 120. This leads to a semi-stable RF generation of 102-112 RF/t. Compare that to a Thermal Expansion boiler connected to a turbine where it gives the full 120 RF/t. So I went ahead and checked at what size it would match... I counted up to a full chunk of fire starters with a tiny 3x3 tank... and now it jumps between 90 mb to 120 mb of steam leading to a semi-stable RF produce between 180 and 240... It appear awfully imbalanced to me to require a FULL chunk of firestarters to even match the ONE block Thermal Expansion dynamo-boiler hybrid.

**Comment by Gaiyamato** [comment]
Yes. the default in the config file is only 80mb Steam per tick **per boiler control unit** 
The whole point is to make modular boilers  that can up-scale very fast. 

You can add more control blocks, pipe in more water and coke and get a better output. 
You can also change the 80mb to something better in the config file.

**Comment by DrUltraLux** [comment]
A new method of steam production is in the work and this issue will be solved in the next update.
